gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/AvroSchemaManager.java

```diff
@@ -110,7 +113,22 @@ public void cleanupTempSchemas() throws IOException {
   }
 
   public static Schema getSchemaFromUrl(Path schemaUrl, FileSystem fs) throws IOException {
```

Comment:
I think the argument should not be a ```Path```. It should be changed to a ```URI```
Add javadoc stating the schemaUrl can be with or without scheme and it defaults to hdfs.